core/src/main/java/org/elasticsearch/common/io/stream/StreamOutput.java

```diff
@@ -454,15 +454,28 @@ public void writeThrowable(Throwable throwable) throws IOException {
     /**
      * Writes a {@link NamedWriteable} to the current stream, by first writing its name and then the object itself
      */
-    public void writeNamedWriteable(NamedWriteable namedWriteable) throws IOException {
+    public void writeNamedWriteable(@SuppressWarnings("rawtypes") NamedWriteable namedWriteable) throws IOException {
         writeString(namedWriteable.getName());
         namedWriteable.writeTo(this);
     }
 
     /**
+     * Writes an optiona {@link NamedWriteable} to the current stream.
+     * */
+    public void writeOptionalNamedWriteable(@SuppressWarnings("rawtypes") NamedWriteable namedWriteable) throws IOException {
```

Comment:
I don't see why the suppress warnings are needed here and in the previous method. Can you clarify that?